### PR TITLE
Supporter+ benefits dropdown

### PR DESCRIPTION
**Pull request by jamesmockett**
## What does this change?

Adds Supporter+ benefits section and toggle button.

## Images

manage thegulocal com_ (1)

manage thegulocal com_ (2)

## Accessibility

-   [x] [Tested with screen reader](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#screen-reader)
-   [x] [Navigable with keyboard](https://github.com/guardian/accessibility/blob/main/people-and-technology/02-physical.md#keyboard)
-   [ ] [Colour contrast passed](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#contrast)
-   [x] [The change doesn't use only colour to convey meaning](https://github.com/guardian/accessibility/blob/main/people-and-technology/03-visual.md#use-of-colour)

### CODE REVIEW

**Comment by jamesmockett** [comment]
[MMA / Account Overview: Add Dropdown for S+ Benefits](https://trello.com/c/p11BjaNo/3638-mma-account-overview-add-dropdown-for-s-benefits)

File: client/components/accountoverview/accountOverviewCardV2.tsx [hunk 1]
```diff
@@ -185,8 +187,105 @@ export const AccountOverviewCardV2 = ({
...
+									<strong>
+										A regular supporter newsletter.
+									</strong>{' '}
+									Get exclusive insight from our newsroom
+								</span>
+							</li>
+							<li>
+								<SvgTickRound size="small" />
+								<span>
+									<strong>Uninterrupted reading.</strong> See
```
Comment (andrewHEguardian) [inline client/components/accountoverview/accountOverviewCardV2.tsx:254]: Some of these are using {' '} to create a space and some are just on the same line. Should we keep it consistent?

File: client/components/payment/nextPaymentDetails.tsx [hunk 2]
```diff
@@ -34,13 +35,14 @@ export const getNextPaymentDetails = (
 			? paidFuturePlans[1]
 			: paidFuturePlans[0];
 	if (isPaidSubscriptionPlan(mainPlan)) {
-		const paymentKey = `Next ${augmentInterval(
+		const paymentInterval =
 			subscription.currentPlans.length !== 0 &&
-				isSixForSix(mainPlan.name) &&
-				planAfterMainPlan
+			isSixForSix(mainPlan.name) &&
```
Comment (andrewHEguardian) [inline client/components/payment/nextPaymentDetails.tsx:40]: Could we maybe split this up? I find it hard to understand what the logic is here

Comment (jamesmockett) [reply client/components/payment/nextPaymentDetails.tsx:40]: Yeah, I completely agree that the logic in here is hard to follow. I think any refactoring of this is probably outside the scope of this PR though. All I've done here is to move the existing logic into a separate variable so we can get the interval (ie. `month`) in isolation from the existing `paymentKey` (ie.`Next monthly payment`)

A lot of the interval logic is tied into whether it's a six for six plan, but I need to find out whether that is still a thing. I don't believe it's something you can sign up for anymore, but there may be readers still on that plan that we need to accommodate.

**Review by andrewHEguardian — APPROVED** [review]
Looks good 👍
